src/features/products/screen/productItem.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Image, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import Icon from 'react-native-vector-icons/Entypo';
+
+import { Product } from 'network/models/product-models';
+
+const ProductItem: React.FunctionComponent<{ item: Product; isLast: boolean }> = ({
+  item,
+  isLast,
+}) => {
+  return (
+    <View style={isLast ? styles.containerLast : styles.container}>
```

Comment:
There is a common pattern to conditionally apply styles, since `style` receives an object or an array of styles you can have one default style to be applied, in this case, `container`.

And, using the validation below if `isLast` is true you will modify or add extra styles for that last item.


```suggestion
    <View style={[styles.container, isLast && styles.containerLast]}>
```

Reply:
Applied it to my next PR, thanks